packages/web/src/common/store/tipping/slice.ts

```diff
@@ -7,6 +7,11 @@ import { User } from 'common/models/User'
 import { BNWei } from 'common/models/Wallet'
 import { TippingState } from 'common/store/tipping/types'
 
+type Thing = PayloadAction<{
+  senderUserId: ID
+  receiverUserId: ID
+}>
```

Comment:
???

Reply:
lol good catch, was using it to figure out the og error, removing